packages/accordion/tests/react/index.js

```diff
@@ -16,18 +16,18 @@ class AccordionWrapper extends React.Component {
 	}
 
 	// let’s change the state in the absence of more complex application code
-	changeAccordion( item ) {
-		this.setState({ [ item ]: !this.state[ item ] });
+	changeAccordion() {
+		this.setState({ closed: !this.state.closed });
 	}
 
 	render() {
 		return (
 			<Fragment>
-				<AUaccordion header="State controlled accordion open" state={ this.state }>
```

Comment:
The reasoning for this, is we know users want to change state that doesn't currently exist on the accordion. I'll have a quick experimentation and see if it needs to be updated.